src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -215,56 +177,56 @@ function PaymentMethodList({
         if (shouldShowAssignedCards) {
             const assignedCards = _.chain(cardList)
                 // Filter by physical, active cards associated with a domain
-                .filter((card) => !card.isVirtual && card.domainName && CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state))
+                .filter((card) => !card.isVirtual && !!card.domainName && CONST.EXPENSIFY_CARD.ACTIVE_STATES.includes(card.state ?? 0))
                 .sortBy((card) => !CardUtils.isExpensifyCard(card.cardID))
                 .value();
```

Comment:
I think you can do it with no lodash

Reply:
What about `sortBy`? I know we could work it around, but I don't think there's a 1:1 standard equivalent

Reply:
Please read the [migration guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#migration-guidelines)!

Reply:
Although the name is a bit weird, actually a standard equivalent is [`toSorted(compareFn)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted). Let's use that!